framework/contracts/account/manager/src/commands.rs

```diff
@@ -924,7 +924,7 @@ pub fn update_internal_config(
 ) -> ManagerResult {
     // deserialize the config action
     let action: InternalConfigAction =
-        from_binary(&config).map_err(|error| ManagerError::InvalidConfigAction { error })?;
+        from_json(config).map_err(|error| ManagerError::InvalidConfigAction { error })?;
```

Comment:
`from_json` only needs value to take `impl AsRef<[u8]>`, was clippy complaining when you were passing it by reference?

Reply:
Oh yeah, I see "needless borrow" nvm